mpisppy/cylinders/spwindow.py

```diff
@@ -0,0 +1,131 @@
+###############################################################################
+# mpi-sppy: MPI-based Stochastic Programming in PYthon
+#
+# Copyright (c) 2025, Lawrence Livermore National Security, LLC, Alliance for
+# Sustainable Energy, LLC, The Regents of the University of California, et al.
+# All rights reserved. Please see the files COPYRIGHT.md and LICENSE.md for
+# full copyright and license information.
+###############################################################################
+
+from mpisppy import MPI
+
+import numpy as np
+import numpy.typing as nptyping
+
+import enum
+
+class Field(enum.IntEnum):
+    SHUTDOWN=-1000
+    NONANT=1
+    DUALS=2
+    OBJECTIVE_BOUNDS=100 # Both inner and outer bounds from the hub. Layout: [OUTER INNER ID]
+    OBJECTIVE_INNER_BOUND=101
+    OBJECTIVE_OUTER_BOUND=102
+    EXPECTED_REDUCED_COST=200
+    SCENARIO_REDUCED_COST=201
+    CROSS_SCENARIO_CUT=300
+    NONANT_COEFS=400
```

Comment:
Because this is used for sending `eta`, which is the recourse cost for scenario s_1 as observed by scenario s_2, it seem like it should be named something else, like `CROSS_SCENARIO_COST` or something similar.

Reply:
Renamed NONANT_COEFS to CROSS_SCENARIO_COST as requested.